### DISCUSSION TITLE
laser stopped connecting

### DISCUSSION THREAD

**Discussion by morilina** [arkypita/LaserGRBL#2482] [Q&A]
My iwecolor laser has stopped being able to connect. I have had it for about a month and everything was working fine, but when I went to use it today it will not connect. I am using the most updated lasergrbl program and have tried everything that I can find to try to get it to connect with no luck. When I try to connect there is a lot of what looks like computer code that comes up and then says error can not connect to your device.

**Comment by morilina** [comment]
![image_2024-12-26_215329423](https://github.com/user-attachments/assets/660a84af-6c95-4a0a-9833-4e891ba546b5)

**Comment by arkypita** [comment]
Given the image, left side that show the messages the machine send, look like machine firmware is corrupt.

You should contact producer or seller for hardwar support
